docs/attributes-registry/vcs.md

```diff
@@ -0,0 +1,27 @@
+| `vcs.repository.change.title`    | string | The human readable title of the change (pull request/merge request). This title is often a brief summary of the change and may get merged in to a ref as the commit summary.                                                                                                                                                                                                                                                                   | `Fixes broken thing`; `feat: add my new feature`; `[chore] update dependency`                                                        | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.ref.commit.hash` | string | The [hash value (see glossary)](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-5.pdf), of the recorded change the ref (branch) is pointing to in the repository. This recorded change is known as the [commit](https://git-scm.com/docs/git-commit). This commit is stored as an object containing a [unique identifier](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefobjectnameaobjectname) which is a SHA. | `9d59409acf479dfa0df1aa568182e43e43df8bbe28d60fcf2bc52e30068802cc`                                                                   | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.ref.name`        | string | The name of the [reference](https://git-scm.com/docs/gitglossary#def_ref) in the repository.                                                                                                                                                                                                                                                                                                                                                   | `my-feature-branch`; `tag-1-test`                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `vcs.repository.ref.type`        | string | The name of the [reference](https://git-scm.com/docs/gitglossary#def_ref) in the repository.                                                                                                                                                                                                                                                                                                                                                   | `my-feature-branch`; `tag-1-test`                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
This looks like a line duplicated from the `vcs.repository.ref.name` line above. While the attribute name changed, the description and examples did not. I'm guessing that this might be what was intended ...

```suggestion
| `vcs.repository.ref.type`        | string | The type of the [reference](https://git-scm.com/docs/gitglossary#def_ref) in the repository.                                                                                                                                                                                                                                                                                                                                                   | `branch`; `tag`                                                                                                    | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Reply:
for sure!